### PR TITLE
Added deprecated static methods at subclass of ManagedTransaction.

### PR DESCRIPTION
**Pull request by sunghwanJo**
## Proposed changes

- Added deprecated static methods at subclass of ManagedTransaction.
- In caver-java 1.0.0, we provided static methods to send transactions for `ValueTransfer`, `Account`, `Cancel`, and `SmartContract` classes. the static methods will removed for extensibility. 

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-java/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-java)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- #5 

## Further comments

-

### CODE REVIEW

File: core/src/main/java/com/klaytn/caver/tx/Account.java [hunk 1]
```diff
@@ -49,4 +49,31 @@ public static Account create(Caver caver, KlayCredentials klayCredentials, int c
 
         return Account.create(caver, transactionManager);
     }
+
+    /**
+     * @deprecated  <p>In caver-java 1.0.0, we provided static methods to send transactions for `ValueTransfer`, `Account`, `Cancel`, and `SmartContract` classes. the static methods were removed for extensibility.</p>
```
Comment (kjhman21) [inline core/src/main/java/com/klaytn/caver/tx/Account.java]:
```suggestion
     * @deprecated  <p>In caver-java 1.0.0, we provided static methods to send transactions for `ValueTransfer`, `Account`, `Cancel`, and `SmartContract` classes. the static methods will be removed.</p>
```

Comment (kjhman21) [reply core/src/main/java/com/klaytn/caver/tx/Account.java]: Please check other tx types.

Comment (sunghwanJo) [reply core/src/main/java/com/klaytn/caver/tx/Account.java]: fixed.  and change '{will be removed in next version}' to 'This deprecated method can be used only for Baobab Testnet'

File: core/src/main/java/com/klaytn/caver/tx/Account.java [hunk 2]
```diff
@@ -49,4 +49,31 @@ public static Account create(Caver caver, KlayCredentials klayCredentials, int c
 
         return Account.create(caver, transactionManager);
     }
+
+    /**
+     * @deprecated  <p>In caver-java 1.0.0, we provided static methods to send transactions for `ValueTransfer`, `Account`, `Cancel`, and `SmartContract` classes. The static methods will be removed. </p>
+     *              <p>This deprecated method can be used only for Baobab Testnet</p>
```
Comment (kjhman21) [inline core/src/main/java/com/klaytn/caver/tx/Account.java]:
```suggestion
     *              <p>This deprecated method can be used only for Baobab Testnet.</p>
```

File: core/src/main/java/com/klaytn/caver/tx/Account.java [hunk 3]
```diff
@@ -49,4 +49,31 @@ public static Account create(Caver caver, KlayCredentials klayCredentials, int c
...
+     *              use {@link #sendUpdateTransaction(AccountUpdateTransaction)} instead like this:
```
Comment (kjhman21) [inline core/src/main/java/com/klaytn/caver/tx/Account.java]:
```suggestion
     *              Use {@link #sendUpdateTransaction(AccountUpdateTransaction)} instead like this:
```

Comment (kjhman21) [reply core/src/main/java/com/klaytn/caver/tx/Account.java]: Shouldn't we have more description after `like this:`? Or, you can remove that phrase simply.

File: core/src/main/java/com/klaytn/caver/tx/Account.java (hunk 2 above)
Comment (sunghwanJo) [reply core/src/main/java/com/klaytn/caver/tx/Account.java]: fixed

File: core/src/main/java/com/klaytn/caver/tx/Account.java (hunk 3 above)
Comment (sunghwanJo) [reply core/src/main/java/com/klaytn/caver/tx/Account.java]: fixed

**Pull request merged**